packages/runtime/test/transport/publicRelationshipTemplateReferences.test.ts

```diff
@@ -0,0 +1,17 @@
+import { RuntimeServiceProvider, TestRuntimeServices } from "../lib";
+
+const serviceProvider = new RuntimeServiceProvider();
+let runtimeServices1: TestRuntimeServices;
+
+beforeAll(async () => {
+    const runtimeServices = await serviceProvider.launch(2);
```

Comment:
It is enough to launch one instead of two runtimeServices instances. :D

I'd prefer to write `runtimeServices = (await serviceProvider.launch(1))[0];` if we only use one runtimeServices instance here and to omit the next line (use variable name `runtimeServices` instead of `runtimeServices1` and declare it above).